symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -5,7 +5,7 @@ framework:
 
         transports:
             # https://symfony.com/doc/current/messenger.html#transports
-            # async: '%env(MESSENGER_TRANSPORT_DSN)%'
+            # async: '%env(MESSENGER_TRANSPORT_DSN)%/messages_exchange'
```

Comment:
Maybe using `?exchange[name]=messages` is more self-explaining. Using the path for the virtual host and exchange is not self-explaining at all.

Reply:
or using the config options directly below the dsn. this would also make sure that options passed in the dsn would keep working.